File: 2219-maximum-number-of-words-found-in-sentences/maximum-number-of-words-found-in-sentences.py
class Solution(object):
    def mostWordsFound(self, sentences):
        """
        :type sentences: List[str]
        :rtype: int
        """

        maxSpace=1

        for i in range(len(sentences)):
            s=sentences[i]
            tempSpace=1
            for i in range(len(s)):
                if(s[i]==" "):
                    tempSpace+=1
               
            if(maxSpace<tempSpace):
                maxSpace=tempSpace
        return maxSpace

        

File: 2219-maximum-number-of-words-found-in-sentences/maximum-number-of-words-found-in-sentences.java
class Solution {
    public int mostWordsFound(String[] sentences) {
        int max =0;
        for(int i=0;i<sentences.length;i++){
            String wordsArray[]=sentences[i].split(" ");
            int length=wordsArray.length;
           // max=Math.max(max,length);
            if(max<length){
                max=length;
            }

        }
        return max;
    }
}